nautilus_core/risk/src/engine/tests.rs

```diff
@@ -0,0 +1,23 @@
+// -------------------------------------------------------------------------------------------------
+//  Copyright (C) 2015-2024 Nautech Systems Pty Ltd. All rights reserved.
+//  https://nautechsystems.io
+//
+//  Licensed under the GNU Lesser General Public License Version 3.0 (the "License");
+//  You may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at https://www.gnu.org/licenses/lgpl-3.0.en.html
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------------------------------
+
+// use rstest::rstest;
+
+// -- TESTS -----------------------------------------------------------------------------------
```

Comment:
Consider keeping the tests in the same module engine/mod.rs module itself since these are unit tests. Also if the engine does not require multiple files consider keeping the logic in a single `engine.rs` file.